velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.cpp

```diff
@@ -14,11 +14,12 @@
  * limitations under the License.
  */
 #include "velox/connectors/hive/storage_adapters/hdfs/HdfsFileSystem.h"
-#include <hdfs/hdfs.h>
+#include <arrow/status.h>
```

Comment:
Since arrow is privately linked with velox_external_hdfs, how is the arrow header visible here?